context/StarknetIdJsProvider.tsx

```diff
@@ -1,21 +1,33 @@
-import React, { FunctionComponent } from "react";
+import { useAccount } from "@starknet-react/core";
+import React, { FunctionComponent, useCallback, useState } from "react";
 import { createContext, useMemo } from "react";
 import { Provider, constants } from "starknet";
 import { StarknetIdNavigator } from "starknetid.js";
+import { hexToDecimal } from "../utils/feltService";
+import { getImgUrl } from "../utils/stringService";
 
 type StarknetIdJsConfig = {
   starknetIdNavigator: StarknetIdNavigator | null;
   provider: Provider | null;
+  identitiesTemp: FullId[];
+  updateIdentityImg: (id: string, imgUrl: string) => void;
+  getPfp: (id: string) => string;
 };
 
 export const StarknetIdJsContext = createContext<StarknetIdJsConfig>({
   starknetIdNavigator: null,
   provider: null,
+  identitiesTemp: [],
+  updateIdentityImg: () => {},
+  getPfp: () => "",
```

Comment:
As we discussed it might be better to use local storage only when the indexer doesn't work